### PR TITLE
Implant chair refactor

### PR DESCRIPTION
**Pull request by AnturK**
Removes some copypasta
Refactors implant chairs. Adds some fun variations.

Merge token : #19002 

dreamseeker_2016-07-12_17-36-39
The spinning cog is recharging notification.

### CODE REVIEW

**Comment by GunHog** [comment]
What are implant chairs?

**Comment by AnturK** [comment]
Old old old code. Put people in press button to implant. Recharges these implants over time.

File: code/game/objects/items/weapons/implants/implantchair.dm [hunk 1]
```diff
@@ -7,141 +7,186 @@
...
-	set src in oview(1)
-	if(usr.stat != 0)
-		return
-	src.go_out(usr)
-	add_fingerprint(usr)
-	return
-
+/obj/machinery/implantchair/brainwash
+	name = "Neural Imprinter"
+	desc = "Used to <s>indoctrinate</s> rehabilitate hardened recidivists."
```
Comment (ChangelingRain) [inline code/game/objects/items/weapons/implants/implantchair.dm]: does `<s>` actually work in byond? hmmmm.

Comment (AnturK) [inline code/game/objects/items/weapons/implants/implantchair.dm]: Yep, well it was faster than text-decorator class. I guess i could change it.

File: code/datums/mind.dm [hunk 2]
```diff
@@ -1319,14 +1319,17 @@
...
-			current << "<B>Objective #[obj_count]</B>: [objective.explanation_text]"
-			obj_count++
+		announce_objectives()
 
 	edit_memory()
 
+/datum/mind/proc/announce_objectives()
+	var/obj_count = 1
+	current << "<span class='notice'>Your current objectives:</span>"
+	for(var/datum/objective/objective in objectives)
```
Comment (RemieRichards) [inline code/datums/mind.dm]: this should be an istypeless forloop.

**Pull request merged**